tachyon/math/finite_fields/BUILD.bazel

```diff
@@ -192,15 +192,15 @@ tachyon_cc_library(
 tachyon_cc_unittest(
     name = "finite_fields_unittests",
     srcs = [
-        # "cubic_extension_field_unittest.cc",
+        "cubic_extension_field_unittest.cc",
         "finite_field_unittest.cc",
-        # "fp12_unittest.cc",
+        "fp12_unittest.cc",
         "fp2_unittest.cc",
-        # "fp6_unittest.cc",
-        # "modulus_unittest.cc",
+        "fp6_unittest.cc",
+        "modulus_unittest.cc",
         "prime_field_base_unittest.cc",
         "prime_field_unittest.cc",
-        # "quadratic_extension_field_unittest.cc",
+        "quadratic_extension_field_unittest.cc",
```

Comment:
Shouldn't this be resolved in commit https://github.com/kroma-network/tachyon/pull/275/commits/e931978314a0c9449875cf3570b16523c62bd67d? (no intermediate changes)